### PR TITLE
refactor(pagination): Tailwindify and follow Figma

### PR DESCRIPTION
**Pull request by julio8a**
**Related Issue:** #1500 #2041

## Summary

- Large updates: 16px font and 16px icons
- Ellipse as text, not icon, follow text size from scale
- Height: Large 44px, Medium 32px, Small 24px

My first time touching TSX files will need review. 

Working from the [Figma/Tailwind components refactor project](https://github.com/Esri/calcite-components/projects/14)

Project card: https://github.com/Esri/calcite-components/projects/14#card-60668015

### CODE REVIEW

File: src/components/calcite-pagination/calcite-pagination.tsx [hunk 1]
```diff
@@ -190,21 +190,21 @@ export class CalcitePagination {
     );
   }
 
-  renderLeftEllipsis(iconScale: this["scale"]): VNode {
+  renderLeftEllipsis(): VNode {
     if (this.total / this.num > maxPagesDisplayed && this.showLeftEllipsis()) {
       return (
         <span class={`${CSS.ellipsis} ${CSS.ellipsisStart}`}>
-          <calcite-icon icon="ellipsis" scale={iconScale} />
+          ...
```
Comment (driskull) [inline src/components/calcite-pagination/calcite-pagination.tsx:197]: Should this use the html entity? `&hellip;`

File: src/components/calcite-pagination/calcite-pagination.tsx [hunk 2]
```diff
@@ -227,12 +226,12 @@ export class CalcitePagination {
           disabled={prevDisabled}
           onClick={this.previousClicked}
         >
-          <calcite-icon dir={dir} flipRtl icon="chevronLeft" scale={iconScale} />
+          <calcite-icon dir={dir} flipRtl icon="chevronLeft" scale="s" />
```
Comment (driskull) [inline src/components/calcite-pagination/calcite-pagination.tsx:229]: Would this be considered a breaking change if we aren't honoring scale anymore?

**Review by driskull — COMMENTED** [review]
comments

File: src/components/calcite-pagination/calcite-pagination.tsx (hunk 2 above)
Comment (julio8a) [reply src/components/calcite-pagination/calcite-pagination.tsx:229]: We are still honoring the scale for the overall size of the pagination component. This just has the size of the chevron icons at small for all sizes, but the text, padding, etc. are following the s, m, l scale.

File: src/components/calcite-pagination/calcite-pagination.tsx (hunk 1 above)
Comment (julio8a) [reply src/components/calcite-pagination/calcite-pagination.tsx]: Thanks! updated this.

**Comment by bstifle** [comment]
OK!

**1.**
So, chevron hover states are using the wrong blue. should be `ui-brand`
image


**2.**
Would be nice to keep the Chevron spacing square:
image
eg. 24x24 hit areas for small, 32x32 for medium, 44x44 for large


Everything else lookin gooooooood

**Review by bstifle — APPROVED** [review]
lookin good!

**Pull request merged**